05-positioning/styles/app.scss

```diff
@@ -4,3 +4,116 @@
 // $fa-font-path: '//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts';
 // @import 'font-awesome';
 @import 'box-sizing';
+
+body {
+  font-family: sans-serif;
+}
+.outter-container {
+  display: flex;
+  justify-content: space-around;
+}
+
+.main-container {
+  border: 1px solid #cecfd5;
+  border-radius: 3%;
+
+  width: 30%;
+  padding: 2.5rem;
+  margin-top: 5rem;
+
+  display: flex;
+  justify-content: space-around;
+  flex-direction: column;
+
+  position: relative;
+  overflow: visible;
+}
+
+.lesson {
+  margin-bottom: 2rem;
+  color: #2db34a;
+  font-size: 1.1rem;
+
+}
+
+.circle {
+  position: absolute;
+  right: -7%;
+  top: 12%;
+  width: 1em;
+  height: 1em;
+
+  display: flex;
+  justify-content: center;
+  align-content: center;
+
+  border-radius: 50%;
+  border: #cecfd5 solid 2px;
+  background: #cecfd5;
+
+  color: white;
+  padding: 3rem;
+  font-size: 3rem;
+}
+
+p {
+  margin: 0;
+  bottom: 22%;
+  left: 36%;
```

Comment:
The other thing you could look at is using the `line-height` center which may allow you to get rid of some extra HTML.